src/Device/jprogOperations.ts

```diff
@@ -30,7 +30,7 @@ const programDeviceWithFw =
             ) {
                 batch.recover('Application', {
                     onTaskBegin: () =>
-                        logger.info(`Device protected, recovering device`),
+                        logger.info(`Device protected; recovering device`),
```

Comment:
Not to sure others will not see this as a typo .... even if this is more correct